gilda/generate_terms.py

```diff
@@ -312,11 +347,18 @@ def get_all_terms():
     terms += generate_mesh_terms()
     terms += generate_uniprot_terms()
     terms += generate_adeft_terms()
+    terms += generate_doid_terms()
+    terms += generate_hp_terms()
+    terms += generate_efo_terms()
     return terms
 
 
-if __name__ == '__main__':
```

Comment:
sticking all of this in a `main()` function reduces conflicts and makes it easier to turn this into an entrypoint later

Reply:
Yes, that's good